src/components/ItemEffects.tsx

```diff
@@ -104,6 +104,10 @@ const ItemEffects = ({ item }: { item: AnnoItem }) => {
             <br />
           </span>
         ))}
+        <strong>{t("trader")}: </strong>
```

Comment:
- can you move the trader section above the "Targets" section for a cleaner UI
- can you hide the trader section when no trader was found